en/19/08.md

```diff
@@ -22,7 +25,7 @@ material:
 
             Zombie[] public zombies;
 
-            // 2. Create a constructor
+            //create constructor
```

Comment:
Can you preserve the numbering of the tasks? I think it makes the course even more user friendly as you can see which comment belongs to which task.
Also, this applies to the other files too